### PR TITLE
`ResultsContainer` + `SaveNewSearchContainer` Design Updates

### PR DESCRIPTION
**Pull request by Icemanbeta**
- Updated `ResultsContainer` to **2018-04-16**
- Updated `SaveNewSearchContainer` to **2018-04-16**

### CODE REVIEW

**Comment by burgwyn** [comment]
apr-16-2018 20-25-48

File: src/Components/SaveNewSearchDialog/SaveNewSearchDialog.jsx [hunk 1]
```diff
@@ -47,6 +47,7 @@ class SaveNewSearchDialog extends Component {
           />
         </FieldSet>
         <div className="saved-search-form-buttons">
+          <button className="usa-button-secondary saved-search-form-secondary-button" onClick={onCancel}>Cancel</button>
```
Comment (mjoyce91) [inline src/Components/SaveNewSearchDialog/SaveNewSearchDialog.jsx]: I feel like Submit should still be the default action when the user presses "enter" while in the text input.

Comment (Icemanbeta) [reply src/Components/SaveNewSearchDialog/SaveNewSearchDialog.jsx]: Oh is the enter tied to the next button or something?

**Comment by Icemanbeta** [comment]
@burgwyn Sorry didn't notice it

image

File: src/Components/SaveNewSearchDialog/SaveNewSearchDialog.jsx (hunk 1 above)
Comment (Icemanbeta) [reply src/Components/SaveNewSearchDialog/SaveNewSearchDialog.jsx]: @mjoyce91 I made some updates and you should be able to hit enter to submit the form, in edit mode it saves to the same search where as in a new state it saves to the new state: 5cb066dc7d6f8e37354241c4eee724aa587703dd

**Pull request merged**